src/services/registration.js

```diff
@@ -0,0 +1,25 @@
+// const regModel = require('../dummy/registrationExample');
+const dbService = require('../services/regCheckDB');
+const users = require('../dummy/userDB');
+
+const createUser = item => new Promise((resolve) => {
+  if (!dbService.CheckUserName(item.username) || !dbService.CheckPassword(item.password)) {
+    resolve('Incorrect registration data');
+  } else if (item.password !== item.confirmPsw) {
+    resolve('Passwords don\'t match');
+  } else if (!dbService.containsUser(item)) {
+    const dbInjection = [];
+    dbInjection.push({
+      username: item.username,
+      password: item.password,
+    });
+    users.userDB.push(dbInjection[0]);
+    resolve(dbInjection[0]);
```

Comment:
you made an array to push a user into it so you can refer that user from the array but you could solve this by creating a user variable.

Reply:
Created the variable